### DISCUSSION TITLE
Confused about mounting my media drive as a subfolder

### DISCUSSION THREAD

**Discussion by billxsand** [photoprism/photoprism#1795] [Q&A] [answered]
Sorry for the noob question but I am very used to Windows and just learning Linux

>Multiple folders can be made accessible by mounting them as subfolders of /photoprism/originals

My media drive is 
>└─sdb1   8:17   0 298.1G  0 part /media/bill/PHOTOS

What should I input to my docker-compose.yml to make this work? and where lol

Hopefully my noobish question will help others down the road

**Comment by graciousgrey** [comment, marked as answer]
You can find example docker-compose files for each operating system here: https://dl.photoprism.org/docker/

You need to replace ~Pictures with the path to your photo library
```
volumes:
      ## The *originals* folder contains your original photo and video files (- "C:/Directory:/photoprism/originals"):
      - "~/Pictures:/photoprism/originals/Pictures"
```

**Reply by billxsand** [reply]
I was overthinking the problem and didn't think to change the variables in 'docker-compose.yml'. Thank you

for reference (future noobs like me) my relevant section now looks like this:

`      ## Multiple folders can be made accessible by mounting them as subfolders of /photoprism/originals:
      - "/media/bill/PHOTOS:/photoprism/originals/PHOTOS"    # [folder 1]:/photoprism/originals/[folder 1]`
